### PR TITLE
Night-vision-removal-from-sg-goggles

### PR DESCRIPTION
**Pull request by KoishiVibe**
# summary

makes M56 headsets not NV. Might actually go back over this and give them very low level NV so you can (dimly) see things in the dark and get jumpscared? 

Either way, you can now readd them to the lockers.

### CODE REVIEW

**Comment by morrowwolf** [comment]
Make a subtype for the no night vision version. Also *you* can add it to the lockers ya know 👀

**Comment by KoishiVibe** [comment]
soon.
todo: test if this variant works
replace in lockers
replace in spawnkits
create spawnkit with regular M56s and with no_nightvision variant

**Comment by KoishiVibe** [comment]
wait fuck no no shit I gotta remove the farsight proc.

**Comment by KoishiVibe** [comment]
Mission complete. This PR should be ready for merging!

File: code/modules/clothing/glasses/night.dm [hunk 1]
```diff
@@ -89,14 +89,15 @@
 /obj/item/clothing/glasses/night/m56_goggles
 	name = "\improper M56 head mounted sight"
 	gender = NEUTER
-	desc = "A headset and goggles system for the M56 Smartgun. Has a low-res short-range imager, allowing for view of terrain."
+	desc = "The headset camera and eyepiece part of the M56 Smartgun system. Connects to the weapon for tracking and detection of targets."
```
Comment (morrowwolf) [inline code/modules/clothing/glasses/night.dm:92]: Undo these changes here

File: code/modules/clothing/glasses/night.dm [hunk 2]
```diff
@@ -198,6 +199,19 @@
 	else
 		button.icon_state = "template"
 
+/obj/item/clothing/glasses/night/m56_goggles/no_nightvision
```
Comment (morrowwolf) [inline code/modules/clothing/glasses/night.dm:202]: Anything that the "parent" has set that is the same here does not need to be redeclared. So like the regular goggles already have gender set to NEUTER and this object inherits that. Please clean this up a bit.

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: code/modules/clothing/glasses/night.dm (hunk 1 above)
Comment (KoishiVibe) [reply code/modules/clothing/glasses/night.dm:92]: Done.

**Review by morrowwolf — APPROVED** [review]

**Pull request merged**